pennylane/devices/default_mixed.py

```diff
@@ -772,7 +758,8 @@ def execute(self, circuit, **kwargs):
     @debug_logger
     def apply(self, operations, rotations=None, **kwargs):
         rotations = rotations or []
-
+        mid_measurements = kwargs.get("mid_measurements", None)
```

Comment:
If executing in `one-shot` mode, there should be an incoming `mid_measurements` dict.